File: Basic/Loop/219432F.cpp
#include <iostream>
using namespace std;

#define intll long long
#define endl '\n'

void solution() {

	int n;
	cin >> n;

	for (int i = 1; i <= 12; i++) {
		cout << n << " * " << i << " = " << i * n << endl;
	}
}

int32_t main() {

	// Test Case
	bool TEST_CASE = !true;

	ios::sync_with_stdio(false) ; cin.tie(0) ;

	int t_c = 1, tt_c = 1;
	if (TEST_CASE) cin >> t_c;

	// cin.ignore(); // In Case getline(cin, s)

	while (t_c--) {
		// cout << "Case " << tt_c++ << ": ";
		solution();
		// if (t_c) cout << '\n';
	}

	return 0;
}

File: Basic/Loop/219432F.go
package main

import (
	"fmt"
)

func solution() {

	var n int
	fmt.Scan(&n)

	for i := 1; i <= 12; i++ {
		fmt.Println(n, "*", i, "=", i*n)
	}
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
